op-challenger/game/scheduler/coordinator.go

```diff
@@ -64,8 +64,12 @@ func (c *coordinator) schedule(ctx context.Context, games []common.Address) erro
 		if j, err := c.createJob(addr); err != nil {
 			errs = append(errs, err)
 		} else if j != nil {
-			jobs = append(jobs, *j)
-			c.m.RecordGameUpdateScheduled()
+			if j.status == types.GameStatusInProgress {
+				jobs = append(jobs, *j)
+				c.m.RecordGameUpdateScheduled()
+			} else {
+				c.logger.Warn("Resolved game update not scheduled", "game", addr, "status", j.status)
```

Comment:
This is normal and very common so probably should just be debug level. Definitely not warn.